runtime/commonMain/src/kotlinx/serialization/json/JsonElement.kt

```diff
@@ -56,6 +56,14 @@ public sealed class JsonElement {
     public val isNull: Boolean
         get() = this === JsonNull
 
+    /**
+     * Convenience method to check whether element represents a [JsonObject] and contains given [key].
+     * Returns false if element is not a [JsonObject].
+     */
+    public operator fun contains(key: String): Boolean {
```

Comment:
KDoc style, it's better not to add "Convenience method" as it neither informative nor helpful.
For example:
```
Checks whether an element is a [JsonObject] and contains the given [key].
```